### PR TITLE
Fix compilation error when using the JDK 21 for Getting Started with Kotlin

### PR DESCRIPTION
**Pull request by Torch3333**
## Description

This fixes the Getting Started with Kotlin that fails to compile when using the JDK 21.

## Related issues and/or PRs

N/A

## Changes made

Update `build.gradle` to fix compilation error when using the JDK 21.

## Checklist

> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

N/A

### CODE REVIEW

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: docs/getting-started-kotlin/build.gradle.kts [hunk 1]
```diff
@@ -25,6 +25,11 @@ tasks.withType<KotlinCompile> {
     kotlinOptions.jvmTarget = "1.8"
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
```
Comment (feeblefakie) [inline docs/getting-started-kotlin/build.gradle.kts:29]: It's not directly related, but should the following be fixed?
https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-by-using-kotlin.md#install-a-jdk

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left one comment. PTAL!

File: docs/getting-started-kotlin/build.gradle.kts (hunk 1 above)
Comment (Torch3333) [reply docs/getting-started-kotlin/build.gradle.kts:29]: [This document was moved to the doc site](https://github.com/scalar-labs/docs-internal-scalardb/blob/main/docs/en-us/getting-started-with-scalardb-by-using-kotlin.mdx) and shows correct JDK requirements. My understanding is that we should not update the document on the ScalarDB repo anymore. 
What do you think ?

**Pull request merged**